app/views/pro/new.html.erb

```diff
@@ -23,4 +25,26 @@
 
     <%= link_to "Sign Up With Facebook", '/auth/facebook?user_type=pro', class: "btn login btn-primary" %>
   </div>
+
+  <div class="ZipRadiusMapContainer">
+    
+    <div id='map'></div>
+        <script>
+            function initMap() {
```

Comment:
Similar to styles, we should keep our JS out of the view. You should be able to move the code inside of the script tags to a JS file in `/app/assets/javascripts`.